app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,37 @@
+class GroupsController < ApplicationController
+  def index
+    @title = 'TRANSACTIONS'
+    @groups = Group.all.where(user_id: current_user.id)
+      .order(created_at: 'desc').includes(:expenses_groups)
+  end
+
+  def show
+    @title = 'DETAILS'
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you if you change this title name from `DETAILS` to `TRANSACTIONS` as the detail page is for TRANSACTIONS. 
**Once you made the above changes, make sure to also update the same names in your `spec_files`**